package.json

```diff
@@ -52,7 +52,7 @@
     "build:release": "grunt build:release",
     "build:webpack-dev": "webpack --config webpack.dev.config.js",
     "build:webpack-release": "webpack --config webpack.release.config.js",
-    "build:webpack-e2e-app": "webpack target/e2e/app/app-using-osw-entry.js target/js/app-bundle.js",
```

Comment:
latest webpack-cli doesn't seem to support output filename parameter, defaulting to `main.js`